src/TestUtils/GcloudWrapper.php

```diff
@@ -131,16 +144,17 @@ public function deploy(
             $this->errorLog('Can not chdir to ' . $this->dir);
             return false;
         }
-        $cmd = "gcloud -q " . self::GCLOUD_APP . " deploy "
+        $beta = $options['beta'] ? 'beta ' : '';
+        $cmd = "gcloud -q " . $beta . self::GCLOUD_APP . " deploy "
             . "--project " . $this->project . " "
             . "--version " . $this->version . " ";
```

Comment:
Then maybe

```php
$cmd = "gcloud -q " . $options['gcloud_channel'] . ' '
 ...
 ...
```
?

Reply:
I used the option name `version` instead